biz.aQute.repository/test/aQute/bnd/repository/maven/pom/provider/PomRepositoryTest.java

```diff
@@ -259,6 +266,71 @@ public void testBndPomRepoRefresh() throws Exception {
 		}
 	}
 
+	/**
+	 * Copies the POM from testdata/pomfiles/simple-nodeps.xml to a temporary file
+	 * which is deleted at the end of the test. This copied POM is added to a Repo
+	 * and updated with a new dependency to test the polling for changes.
+	 * Furthermore a remote POM is added to the config to check run through the
+	 * stale-check
+	 */
+	public void testBndPomRepoFilePolling() throws Exception {
+		Path path = Paths.get("testdata/pomrepo/simple-nodeps-copy.xml");
+		Files.copy(Paths.get("testdata/pomrepo/simple-nodeps.xml"), path, StandardCopyOption.REPLACE_EXISTING);
+
+		BndPomRepository bpr = new BndPomRepository();
+		Workspace w = Workspace.createStandaloneWorkspace(new Processor(), tmp.toURI());
+		w.setBase(tmp);
+		bpr.setRegistry(w);
+
+		Map<String,String> config = new HashMap<>();
+		config.put("pom",
+				path.toString()
+						+ ",https://repo1.maven.org/maven2/org/apache/felix/org.apache.felix.gogo.shell/0.12.0/org.apache.felix.gogo.shell-0.12.0.pom");
+		config.put("snapshotUrls", "https://repo1.maven.org/maven2/");
+		config.put("releaseUrls", "https://repo1.maven.org/maven2/");
+		config.put("name", "test");
+		config.put("pollTime", "300");
```

Comment:
This is now `"poll.time`. The property is described as poll time in seconds. 

See the following example of poll testing, It uses a plugin to detect that refresh has been called. You will need to add the RepositoryListenerPlugin to the workspace since the workspace is set as the registry for bpr.

 https://github.com/bndtools/bnd/blob/5d8500fd174eb7ec4228f383baba910d4d5573f8/biz.aQute.repository/test/aQute/bnd/repository/osgi/OSGiRepositoryTest.java#L125-L192